code/modules/clothing/head/misc_special.dm

```diff
@@ -305,17 +305,23 @@
 	clothing_flags = EFFECT_HAT | SNUG_FIT
 	armor = list("melee" = 0, "bullet" = 0, "laser" = -5,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = -5, "fire" = 0, "acid" = 0)
 	equip_delay_other = 140
-	var/datum/brain_trauma/mild/phobia/conspiracies/paranoia
+	var/datum/brain_trauma/mild/phobia/conspiraciesandlizards/paranoia
+	var/datum/brain_trauma/mild/phobia/conspiracies/paranoia_lizard
 
 /obj/item/clothing/head/foilhat/equipped(mob/living/carbon/human/user, slot)
 	..()
-	if(slot == SLOT_HEAD)
-		user.sec_hud_set_implants()
+	if(!is_species(/datum/species/lizard))
+		if(paranoia_lizard)
+			QDEL_NULL(paranoia_lizard)
+		paranoia_lizard = new()
+		paranoia_lizard.clonable = FALSE
+		user.gain_trauma(paranoia_lizard, TRAUMA_RESILIENCE_MAGIC)
+		to_chat(user, "<span class='warning'>As you don the foiled hat, an entire world of conspiracy theories and seemingly insane ideas suddenly rush into your mind. What you once thought unbelievable suddenly seems.. undeniable. Everything is connected and nothing happens just by accident. You know too much and now they're out to get you. </span>")
+	else
 		if(paranoia)
 			QDEL_NULL(paranoia)
 		paranoia = new()
 		paranoia.clonable = FALSE
```

Comment:
This whole section has a lot of repeated code, I feel like if conspirisies without the lizards was a subtype of conspiricies you could make the whole thing a lot less and just have
```
if(paranoia)
		QDEL_NULL(paranoia)
if(is_species(/datum/species/lizard))
		paranoia = new /datum/brain_trauma/mild/phobia/conspiracies/nolizards()
else
		paranoia = new ()
paranoia_lizard.clonable = FALSE
		user.gain_trauma(paranoia_lizard, TRAUMA_RESILIENCE_MAGIC)
		to_chat(user, "<span class='warning'>As you don the foiled hat, an entire world of conspiracy theories and seemingly insane ideas suddenly rush into your mind. What you once thought unbelievable suddenly seems.. undeniable. Everything is connected and nothing happens just by accident. You know too much and now they're out to get you. </span>")
```